src/controllers/Interactions.ts

```diff
@@ -0,0 +1,103 @@
+import { Client, Interaction, TextChannel } from "discord.js";
+import { DateTime } from "luxon";
+import LeaderboardRepository from "../repositories/LeaderboardRepository";
+import QueueRepository from "../repositories/QueueRepository";
+import { BallChaser } from "../types/common";
+import getDiscordChannelById from "../utils/getDiscordChannelById";
+import getEnvVariable from "../utils/getEnvVariable";
+import MessageBuilder from "../repositories/helpers/MessageBuilder";
+
+const NormClient = new Client({ intents: "GUILDS" });
+const queueChannelId = getEnvVariable("queue_channel_id");
+
+
+export async function buttonEmbeds(queueChannel: TextChannel): Promise<void> {
```

Comment:
Is this just sending the initial queue message on start? Can we give this function a better name to reflect that, please.

Reply:
Fixed in next commit